pkg/client/client.go

```diff
@@ -1004,9 +1017,35 @@ func monitorKeepTerminatedEvent(
 					members,
 				)
 
-				tss.BroadcastRecoveryAddress(
+				if err != nil {
```

Comment:
We should consider handling failures in a retry loop, e.g. as in https://github.com/keep-network/keep-ecdsa/blob/9f5826bacf329ee119666cfe70eb62f947a4afcb/pkg/client/client.go#L651